### DISCUSSION TITLE
Logging converting untitiled notes when processing .enex files

### DISCUSSION THREAD

**Discussion by boardtc** [akosbalasko/yarle#506] [General]
A have come back to Yarle after another 18 months sticking with Evernote hoping for a clean exit. I redownloaded the current windows version.

I exported 2 notebooks from Evernote with all notes having a title.

I tested converting both .enex files with Yarle, sticking with the default option except adding .md to the file names. One of the notebooks had 14 notes and the other 49, both conversions were done in single batches and both appeared to complete normally.

Opening the output in Obsidian I found Untitled notes, the first had 2 and the 2nd 1.

I don't know if there is any way to view an enex file to see it's correctly formatted? That said looking at the yarle log for the first conversion it says:

    Converting note "Untitled Note"...
    relative resource work dir: ./_resources/Untitled_Note.1.resources
    Converting note "Untitled Note"...

Does this point to the evernote .enex export getting it wrong? Is that a known thing?

I have quite a lot of notebooks/notes in my bid to escape Evernote and I would prefer not to have to check everything, but perhaps that is needed? Any advice appreciated!

**Comment by akosbalasko** [comment]
Hi @boardtc ,

Nope, it is normal, webclips used to be exported to enex files with a title called "Untitled Note".

Best Regards,
akos

**Comment by boardtc** [comment]
To close this one out after some debugging it's an issue with evernote, more info here https://discussion.evernote.com/forums/topic/130330-untitled-notes-in-enex-import/

**Reply by akosbalasko** [reply]
Great thanks a lot, I'll keep my eyes on this thread!
